controllers/storagecluster/cephfilesystem.go

```diff
@@ -79,6 +79,9 @@ func (r *StorageClusterReconciler) newCephFilesystemInstances(initStorageCluster
 		}
 	}
 
+	// set device class for metadata pool from the default data pool
```

Comment:
```suggestion
	// set device class for metadata pool from the default metadata pool
```

Reply:
By this comment I meant we are fetching the deviceClass to put in the metadata pool from the 1st data pool in the CephFS data pools list.